src/main/scala/com/typesafe/sbt/web/SbtWeb.scala

```diff
@@ -529,16 +540,21 @@ object SbtWeb extends AutoPlugin {
    * @return
    *   the (possibly) deduplicated mappings
    */
-  def deduplicateMappings(mappings: Seq[PathMapping], deduplicators: Seq[Deduplicator]): Seq[PathMapping] = {
+  def deduplicateMappings(
+      mappings: Seq[PathMapping],
+      deduplicators: Seq[Deduplicator],
+      conv: FileConverter
```

Comment:
nitpick, but you forgot to add the `@param` to the javadocs.

Reply:
Added it here, and in a couple of other places as well.